src/seedu/addressbook/commands/AddCommand.java

```diff
@@ -66,5 +66,6 @@ public CommandResult execute() {
             return new CommandResult(MESSAGE_DUPLICATE_PERSON);
         }
     }
-
+    @Override
+    public boolean isMutating() { return true; }
```

Comment:
Coding standard violation. Please check how are methods with a single line of code written.